contrib/test/integration/vars.yml

```diff
@@ -3,6 +3,24 @@
 # When swap setup is necessary, make it this size
 swapfileGB: 8
 
+# Path to encode into /etc/environment on all hosts
+go_path: "/go"
```

Comment:
You've changed this from a value that gets inferred to one that is declarative. While I agree that it is better to be declarative, some of the bits that make the jobs work today expect `$GOPATH` to exist and be at `/data` ... so I am not entirely certain this will work.

Reply:
Thanks for the feedback.  Darn.  So, part of changing away from squirreling them away in files, was to explicitly plaster their use up-front-and-center by all the ``environment`` directives (in the playbook).  However, since you're concerned...maybe I should add a comment to clarify?

Alternatively, what if I moved the big ``environment_variables`` block to right at the beginning of ``vars.yml``?

How to make this pattern easier to follow for Ansible-n00b's?